packages/core/src/routes/connector.ts

```diff
@@ -146,8 +148,22 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
         validateConfig(body.config);
       }
 
+      if (metadata.isStandard === true) {
+        const databaseMetadataGuard = configurableConnectorMetadataGuard.required();
+        const result = databaseMetadataGuard.safeParse(body.metadata);
```

Comment:
To align with `ConnectorResponse`, better remove "metadata" and spread its elements.

Reply:
Current update-related APIs response body is not aligned with ConnectorResponse type, maybe fix this later.